core-bundle/assets/scripts/choices.js

```diff
@@ -0,0 +1,58 @@
+(function () {
+    'use strict';
+
+    const initializedChoices = new WeakMap();
+
+    const init = (select) => {
+        // Check if this select has already been initialized
+        if (initializedChoices.has(select)) {
+            return;
+        }
+
+        initializedChoices.set(select, true);
+
+        new Choices(select, {
+            shouldSort: false,
+            duplicateItemsAllowed: false,
+            allowHTML: false,
+            removeItemButton: true,
+            searchEnabled: select.options.length > 7,
+            classNames: {
+                containerOuter: ['choices', ...Array.from(select.classList)],
+                flippedState: ''
+            },
+            fuseOptions: {
+                threshold: 0.4,
+            },
+            callbackOnInit: () => {
+                const choices = select.closest('.choices')?.querySelector('.choices__list--dropdown > .choices__list');
+
+                if (choices && select.dataset.placeholder) {
+                    choices.dataset.placeholder = select.dataset.placeholder;
+                }
+            },
+            loadingText: Contao.lang.loading,
+            noResultsText: Contao.lang.noResults,
+            noChoicesText: Contao.lang.noChoices,
+            itemSelectText: Contao.lang.pressToSelect,
+            uniqueItemText: Contao.lang.onlyUnique,
+            customAddItemText: Contao.lang.onlyConditions
```

Comment:
There are more translations in the original script:

```js
addItemText: function (value) { return "Press Enter to add <b>\"".concat(value, "\"</b>"); },
removeItemIconText: function () { return "Remove item"; },
removeItemLabelText: function (value) { return "Remove item: ".concat(value); },
maxItemText: function (maxItemCount) { return "Only ".concat(maxItemCount, " values can be added"); },
valueComparer: function (value1, value2) { return value1 === value2; },
```

Should we translate them as well?

Reply:
Good catch, we need to that for the multiselect